kfdefs/overlays/moc/smaug/opf-jupyterhub/blackbox-exporter.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: blackbox-exporter
+spec:
+  port:
+    targetPort: 9115-tcp
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: blackbox-exporter
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: blackbox-exporter
+    prometheus.io/scrape: "true"
+    prometheus.io/joblabel: "blackbox-metrics"
+  name: blackbox-exporter
+spec:
+  ports:
+  - name: 9115-tcp
+    port: 9115
+    protocol: TCP
+    targetPort: 9115
+  selector:
+    deployment: blackbox-exporter
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: blackbox
+data:
+  blackbox.yml: |
+    modules:
+      http_2xx:
+        prober: http
+        timeout: 10s
+        http:
+          valid_http_versions: ["HTTP/1.1", "HTTP/2.0"]
+          method: GET
+          valid_status_codes: [200, 201, 202, 203, 204, 205, 206, 207, 208, 403]
+          preferred_ip_protocol: "ip4"
+          tls_config:
```

Comment:
remove this tlsConfig since it is referencing to the rh internal cert

Reply:
done